### PR TITLE
Getting started experience improvements

### PR DESCRIPTION
**Pull request by BrynCooke**
Updated the docs where they were out of date.
Removed the `Error:` prefix from our help message.
Add install script.

The install script should be a stopgap until Rover has the ability to download and manage versions of the router.

### CODE REVIEW

File: apollo-router/src/executable.rs [hunk 1]
```diff
@@ -261,8 +260,11 @@ APOLLO ROUTER v{version}
 
     $ ./router --supergraph starstuff.graphql
 
-    "#,
-            ));
+    "#
+            );
+            // The reason to exit here is that anyhow will put an `Error:` prefix on the message.
+            // <aybe this is slightly less clean, but it looks better for the user
```
Comment (bnjjj) [inline apollo-router/src/executable.rs:266]:
```suggestion
            // Maybe this is slightly less clean, but it looks better for the user
```

Comment (bnjjj) [reply apollo-router/src/executable.rs:266]: Even if we `return Ok(());` ?

Comment (BrynCooke) [reply apollo-router/src/executable.rs:266]: I guess we need to debate if the help message is an OK or Error.
It can wait til Monday :)

Comment (bnjjj) [reply apollo-router/src/executable.rs:266]: My main concern with using process::exit is when we are writing tests. I recently had this kind of situation (for plugin shutdown) where we basically call process::exit when something goes wrong and in the test we just have no clue of what's going on. But yes I think it's less problematic in this case as it's directly inside the main function

Comment (BrynCooke) [reply apollo-router/src/executable.rs:266]: I guess we could catch the error in the main function and translate it?

Comment (BrynCooke) [reply apollo-router/src/executable.rs]: Moved

**Pull request merged**